sentry-samples/sentry-samples-servlet/src/main/java/io/sentry/samples/servlet/SentryInitializer.java

```diff
@@ -1,57 +1,54 @@
 package io.sentry.samples.servlet;
 
 import io.sentry.Sentry;
-
+import java.util.Set;
 import javax.servlet.ServletContainerInitializer;
 import javax.servlet.ServletContext;
 import javax.servlet.ServletException;
-import java.util.Set;
 
-/**
- * Initializes Sentry.
- */
+/** Initializes Sentry. */
```

Comment:
M: looks like you've missed the `build or spotlessApply` locally?

Reply:
def didn't bother with that, just wanted to see if it works